arches_lingo/serializers.py

```diff
@@ -41,6 +42,14 @@ def validate_appellative_status(self, data):
 
         return data
 
+    @staticmethod
+    def get_reference_object(data) -> Reference:
+        # TODO: serializer should just do this itself, only waiting to tackle:
```

Comment:
Glad we got the ball rolling on this, now this is moving up higher on my priority list to find a sane generic place to do these transformations.